pdns/dnsdistdist/docs/reference/ebpf.rst

```diff
@@ -81,15 +81,16 @@ These are all the functions, objects and methods related to the :doc:`../advance
 
     :param ComboAddress address: The address to block
 
-  .. method:: BPFFilter:blockRange(Netmask [, force=false])
+  .. method:: BPFFilter:addRangeRule(Netmask , action [, force=false])
 
   .. versionchanged:: 1.8.0
 
     Block all IP addresses in this range. 
 
     DNSDist eBPF code first checks if an exact IP match is found, then if a range matches, and finally if a DNSName does.
 
-    :param string Netmask: The ip range to block
+    :param string Netmask: The ip range to block or unblock
+    :param int action: set ``action``  to ``0`` to unblock a range, set ``action`` to ``1`` to block a range.
```

Comment:
I wonder if something like "set ``action`` to ``0`` to allow a range, set ``action`` to ``1`` to block a range." would not be clearer?
Shouldn't we document the "2"/"truncate" option there?

Reply:
Oh, yeah, I guess that makes sense.